ui/raidboss/timeline.js

```diff
@@ -808,14 +855,17 @@ export class TimelineUI {
     }
 
     // Adding a timer with the same id immediately removes the previous.
-    if (this.activeBars[e.id]) {
-      const div = this.activeBars[e.id].parentNode;
-      div.parentNode.removeChild(div);
+    const activeBar = this.activeBars[e.id];
+    if (activeBar) {
+      const div = activeBar.parentNode;
+      if (div && div.parentNode)
+        div.parentNode.removeChild(div);
     }
 
     div.style.order = e.sortKey;
     div.id = e.id;
-    this.timerlist.appendChild(div);
+    if (this.timerlist)
+      this.timerlist.appendChild(div);
```

Comment:
```suggestion
    this.timerlist?.appendChild(div);
```

Reply:
WAIT.  Sorry for this suggestion, but I didn't think you could use `?.` in JavaScript yet until OverlayPlugin updates its CEF version.  How is this not complaining?

Reply:
It might be complaining; I haven't loaded the overlay since I initially made the PR since I'm just addressing comments right now

Reply:
Committed and reverted.